### PR TITLE
Move the FSF up on the page

### PR DESCRIPTION
**Pull request by evandandrea**
Not convinced by how it looks at the top, but this PR will at least give us something to debate around.

Screenshot 2019-03-25 at 15 00 41

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -43,6 +43,20 @@ <h2>The app store for Linux</h2>
   </div>
 </div>
 
+<div class="p-strip">
+    <div class="row">
```
Comment (bartaz) [inline templates/index.html]: Seems like indentation got a bit shifted after copy/pasting.

**Comment by therealjuan** [comment]
I wonder if we could just leave the publishers still at the very top and replace the FSF module with something more along the lines of:

image

**Comment by matthewpaulthomas** [comment]
That’s roughly what I had in mind for when the number of languages/frameworks became too wide for a single row of icons. (I was surprised when the icons were spread out so much that only 12 wrapped to two rows.)

I think a greater distinction between the tabs and their contents would help people understand where they start and stop and how they interact. (This would be even more important if the section is moved away from the bottom of the page.) A couple of extra hairlines would be enough:

front-page-languages-frameworks-vertical

**Comment by evandandrea** [comment]
>I wonder if we could just leave the publishers still at the very top and replace the FSF module with something more along the lines of…

Where would the FSF module sit on the page?

**Comment by evandandrea** [comment]
I'm not convinced by having the example yaml on a hidden-by-default tab. There are differences in benefits to users of different languages –bundling dependencies in C/C++ is a lifesaver, but solved by the language in Java and Go– however, there is enough commonality between the [listed benefits](https://snapcraft.io/first-snap) that I worry each language tab will feel too similar.

Does someone who's not written as many snaps have an opinion? What would motivate you to continue?

**Comment by therealjuan** [comment]
I was referring to something like this @evandandrea:

FSF Updated

We should implement this and improve the layout afterwards.

File: templates/index.html (hunk 1 above)
Comment (evandandrea) [reply templates/index.html]: Sorry about that. Fixed as 81c16b1

**Comment by evandandrea** [comment]
>We should implement this and improve the layout afterwards.

Sure, 81c16b1 puts it beneath the major publishers.

**Pull request merged**